0-bootstrap/modules/cb-private-pool/network.tf

```
@@ -40,6 +40,18 @@ module "peered_network" {
 
 }
 
+resource "google_dns_policy" "default_policy" {
+  count = var.private_worker_pool.create_peered_network ? 1 : 0
+
+  project                   = var.project_id
+  name                      = "dp-b-peer-default-policy"
```

Would it be better to signify this is for private pools?
```suggestion
  name                      = "dp-b-cbpools-default-policy"
```

---

fixed